From 3ce0fddca0c79fdc2ea0a35bcb57c738140583dd Mon Sep 17 00:00:00 2001
From: oceanicc <oceanic@mail.com>
Date: Wed, 9 Jul 2025 03:14:43 +0530
Subject: [PATCH] pywal colors mew

---
 config.def.h | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/config.def.h b/config.def.h
index 3cfa4da..7d78b2b 100644
--- a/config.def.h
+++ b/config.def.h
@@ -1,14 +1,15 @@
 /* See LICENSE file for copyright and license details. */
 /* Default settings; can be overriden by command line. */
+#include "colors.h"
 
 static int top              = 1;                       /* -b option; if 0, appear at bottom */
-static const char *fonts[]  = { "monospace:size=10" }; /* -f option overrides fonts[0] */
+static const char *fonts[]  = { "monospace:size=11" }; /* -f option overrides fonts[0] */
 static const char *prompt   = NULL;                    /* -p option; prompt to the left of input field */
 static uint32_t colors[][2] = {
- /*               fg         bg          */
- [SchemeNorm] = { 0xbbbbbbff, 0x222222ff },
- [SchemeSel]  = { 0xeeeeeeff, 0x005577ff },
- [SchemeOut]  = { 0x000000ff, 0x00ffffff },
+  /*               fg           bg          */
+  [SchemeNorm] = { 0xbbbbbbff,  color0},
+  [SchemeSel]  = { color0,      color2},
+  [SchemeOut]  = { 0x000000ff,  0x00ffffff },
 };
 
 /* -m option; if provided, use that output instead of default output */
-- 
2.50.0

